changelogs/current.yaml

```diff
@@ -2,6 +2,14 @@ date: Pending
 
 behavior_changes:
 # *Changes that are expected to cause an incompatibility if applicable; deployment changes are likely required*
+- area: eds
```

Comment:
Should https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol#resource-warming also be updated?

Reply:
Good idea, thanks. Done.